tests/test_tensorrt_preprocess.py

```diff
@@ -0,0 +1,74 @@
+import os
```

Comment:
Could we move this test file to the `test_ops` directory for better organization?

Reply:
Done.